publish.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# Install dependencies and build our project
+./installDependencies.sh
+./buildAll.sh
+
+# Publish the extension. By default, VSCE will use the $VSCE_PAT
```

Comment:
Is this something we need to configure?

Reply:
Not sure what you mean by configure. The environment variable for publisher token?

Reply:
To clarify: we will use this script in a GH Actions workflow where we can provide the publisher token securely. This cannot be leaked (would give access to Trelent's vsce publisher account) so we dont configure it code-side.

Reply:
ah I see, alright LGTM then!